### PR TITLE
Update protocols.md

### PR DESCRIPTION
**Pull request by gemantzu**
typo

### CODE REVIEW

**Comment by nscyclone** [comment]
I don't think that this is a typo. `It is worth noting` means that there is something the reader should pay attention to.
Maybe [this](http://www.linguee.gr/%CE%B1%CE%B3%CE%B3%CE%BB%CE%B9%CE%BA%CE%AC-%CE%B5%CE%BB%CE%BB%CE%B7%CE%BD%CE%B9%CE%BA%CE%AC/%CE%BC%CE%B5%CF%84%CE%AC%CF%86%CF%81%CE%B1%CF%83%CE%B7/it+is+worth+noting.html) could be helpful.

**Comment by gemantzu** [comment]
Maybe I got it wrong then? Is it intended to be `It is worth nothing`?

**Comment by nscyclone** [comment]
I suppose that `It is worth noting` is the right option in this case.
For me, actually, `It is worth to note` sounds more usual and as a bonus it will not lead to a misread.

**Comment by gemantzu** [comment]
Yes, that's what I got while reading this (actually, I didn't even notice that it said `nothing` at first) and that's why I proposed the change (and that's how I translated it in Greek)

File: lessons/advanced/protocols.md [hunk 1]
```diff
@@ -110,6 +110,6 @@ iex> to_atom(%{foo: "bar"})
 :foo
 ```
 
-It is worth nothing that although underneath structs are Maps, but they do not share protocol implementations with Maps. They are not enumerable, they cannot be accessed.
+It is worth noting that although underneath structs are Maps, but they do not share protocol implementations with Maps. They are not enumerable, they cannot be accessed.
```
Comment (doomspork) [inline lessons/advanced/protocols.md]: @gemantzu / @nscyclone how about something like:  "It is worth it to note, that although..."

Comment (nscyclone) [inline lessons/advanced/protocols.md]: @doomspork I didn't have any issues with understanding that sentence but as I mentioned above this option could prevent potential misreading that led to creation of this issue.

Comment (gemantzu) [inline lessons/advanced/protocols.md]: I didn't have issues as well, I just thought that it was a typo, hence the issue. Both of your suggestions seem better to me than the original.

Comment (doomspork) [inline lessons/advanced/protocols.md]: @gemantzu if you want to make this change we can get this merged in 👍

**Comment by doomspork** [comment]
Thank you @gemantzu and @nscyclone!

**Pull request merged**